.travis.yml

```diff
@@ -28,17 +33,25 @@ compiler:
 env:
   matrix:
   - Q_OR_C_MAKE=cmake
+    QT_VERSION="59" # actually 5.9
   - Q_OR_C_MAKE=qmake
+    QT_VERSION="59" # actually 5.9
   global:
   - secure: VFI3UCiDrp47WTcUhsatdQvvWg+3gk00eBMZgSOXXKY5+hk+NOX7bOFcIM5t9nlZDbpHDr10SFTuUOw+PeWmLpFO06Zrjg86M9jm9WS4i8Cs9hfxoT6H4isXlR1vubX2LmNlHyzg8WtdNanlsufgecyaGksJxr7tVhG/cWyD6yo=
 matrix:
   exclude:
   - os: osx
     compiler: gcc
+  include:
+  - os: linux
+    compiler: gcc
+    env: 
+    - Q_OR_C_MAKE="cmake"
```

Comment:
No need for the `"` I think - they are appearing in "Config" the one specific build job that this bit creates which is different from the others - it hasn't started building yet so cannot tell whether it still works...
![travis_snapshot_1](https://user-images.githubusercontent.com/6163092/27692579-f967aba6-5cde-11e7-8382-b43c84c0bfc4.png)